sessionState/README.md

```diff
@@ -0,0 +1,55 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Geode Session State Example using Tomcat
+
+This is a simple example which demonstrates how to setup and use the Session Management Module for Tomcat.
+
+This example assumes you have Geode and Java installed. It also assumes you have local copy of Tomcat Downloaded. 
+It was designed and tested with Geode 1.9.0 and Tomcat 9, and while the session features will work with other combinations
+you may need to make some changes to the setup if you're using other versions.
```

Comment:
I wonder if at least we could link to the Geode documentation for the latest libs requirements since that seems to have been what changes in the past?  The list here:
https://geode.apache.org/docs/guide/11/tools_modules/http_session_mgmt/tomcat_installing_the_module.html